### PR TITLE
Change peer dependency to `babel-plugin-emotion` for emotion `^8.0.2` update

### PR DESCRIPTION
**Pull request by fanderzon**
These are the changes I had to make to get emotion 8.0.2 to work with `react-app-rewire-emotion`.

### CODE REVIEW

File: packages/react-app-rewire-emotion/package.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
   "author": "osdevisnot <osdevisnot@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "emotion": "^7.2.2"
+    "emotion": "^8.0.2"
```
Comment (osdevisnot) [inline packages/react-app-rewire-emotion/package.json]: with the change in namespace, this peerDependency should really be on babel-plugin-emotion

Comment (fanderzon) [reply packages/react-app-rewire-emotion/package.json]: Good point! Updated. 🙂

**Comment by osdevisnot** [comment]
Published as - react-app-rewire-emotion@3.0.0

**Comment by tkh44** [comment]
Thanks @fanderzon!

**Pull request merged**